tests/tests.bats

```diff
@@ -187,3 +187,27 @@ DOCKER_PLUGIN_DEFAULT_ARG="/usr/sbin/sshd -D -p 22"
   run docker run --entrypoint sh --rm "${SUT_IMAGE}" -c 'locale charmap'
   assert_equal "${output}" "UTF-8"
 }
+
+
+@test "[${SUT_IMAGE}] image has ssh client, less and patch installed and in the PATH" {
+  local test_container_name=${AGENT_CONTAINER}-bash-java
+  clean_test_container "${test_container_name}"
+  docker run --name="${test_container_name}" --name="${test_container_name}" "${docker_run_opts[@]}" "${PUBLIC_SSH_KEY}"
+
+  run docker exec "${test_container_name}" sh -c "command -v ssh"
+  assert_success
+  run docker exec "${test_container_name}" ssh -V
+  assert_success
+
+  run docker exec "${test_container_name}" sh -c "command -v less"
+  assert_success
+  run docker exec "${test_container_name}" less -V
+  assert_success
+
+  run docker exec "${test_container_name}" sh -c "command -v patch"
+  assert_success
+  run docker exec "${test_container_name}" patch -v
+  assert_success
+
+  clean_test_container "${test_container_name}"
+}
```

Comment:
This test is good because it underlines issues that we currently have on the Alpine  image.

But the PR starts to grow: what do you think about splitting the work on different PRs to make it easier and smoother to review and deliver?

- 1 PR should be to test for the presence of SSH + git (`chore(linux) add a test to ensure that git and the SSH client are installed
- 1 PR for adding `less` and `patch` (with the test updated)
- 1

Reply:
Thanks for the proposal.
I will split the PR into two different ones, addressing specific/different subjects.

I don't have the right "muscle memory" yet when it comes to small PRs that suddenly grow and address a related subject or a totally different one. 🤷 

A time will come when I'll do it "correctly" without even thinking of it. 👍